.gitignore

```diff
@@ -81,6 +81,7 @@ target/
 .venv
 ENV/
 venv/
+os/
```

Comment:
What do we need that for?

Reply:
Hey, pls check.. I removed the unnecessary change of this